### ISSUE TITLE
SQS example fails

### ISSUE THREAD

**Issue by svekars** [pachyderm/pachyderm#4334]
**What happened?**:
I've tried to follow the steps in the [SQS example](https://github.com/pachyderm/pachyderm/tree/master/examples/spouts/SQS-S3) and failed to get a functional pipeline. I was able to get pipeline created, but it never runs a single job, and I get the following message in the logs:

```
Traceback (most recent call last):
  File "/sqs-spout.py", line 173, in <module>
    main()
  File "/sqs-spout.py", line 124, in main
    msgs = retrieve_sqs_messages(args.sqs_queue_url, args.aws_access_key_id, args.aws_secret_access_key, args.aws_region, num_messages)
  File "/sqs-spout.py", line 57, in retrieve_sqs_messages
    return msgs['Messages']
KeyError: 'Messages'
```
This makes me think that there might be something wrong with the Python script.
**What you expected to happen?**:

A successful pipeline run. 
**How to reproduce it (as minimally and precisely as possible)?**:

1. Create an S3 bucket
1. Create an SNS topic and a standard SQS queue as described in The [SNS/SQS example walktrhough](https://docs.aws.amazon.com/AmazonS3/latest/dev/ways-to-add-notification-config-to-bucket.html).
1. Create pipeline by using sqs-spout.json.

**Anything else we need to know?**:
@pappasilenus , if you want to check out my setup, here it is:

S3 bucket: svekars1 (us-west-1)
SNS topic: svekars-topic (us-west-1)
SQS queue: svekars1-SQS
The only modification done to the `sqs-spout.py` script is `print(msgs)` on line 53, which I'm not adding here. 

**Environment?**:
- Kubernetes version (use `kubectl version`): EKS
- Pachyderm CLI and pachd server version (use `pachctl version`): 1.9.8
- Cloud provider (e.g. aws, azure, gke) or local deployment (e.g. minikube vs. dockerized k8s): AWS
- OS (e.g. from /etc/os-release): macOS
- Others:

**Comment by svekars** [comment]
[sqs-spout.txt](https://github.com/pachyderm/pachyderm/files/3838340/sqs-spout.txt)

**Comment by pappasilenus** [comment]
Fixed in https://github.com/pachyderm/pachyderm/pull/4336, I hope.

**Comment by svekars** [comment]
resolved by #4336
